src/app/shared/components/header/header.component.scss

```diff
@@ -0,0 +1,55 @@
+header {
+    padding: 10px 0;
+    border-bottom: 1px solid grey;
+    position: fixed;
+    width: 100%;
+    background: white;
+    z-index: 100;
+    .main-links {
+        li {
+            list-style-type: none;
+            float: left;
+            margin: 0 15px;
+        }
+    }
+    a img {
+        width: 50px;
+    }
+    .link-part .menu-links {
+        line-height: 42px;
+    }
+    .userPart {
+        margin-right: 15px;
+    }
+    .loggedIn {
```

Comment:
you can nest these to avoid repeating .loggedIn so many times

Reply:
Done